packages/react-native/ReactAndroid/src/main/jni/react/fabric/JFabricUIManager.cpp

```diff
@@ -13,8 +13,8 @@ namespace facebook::react {
 
 Binding* JFabricUIManager::getBinding() {
   static const auto bindingField =
-      javaClassStatic()->getField<Binding::javaobject>("mBinding");
+      javaClassStatic()->getField<JBinding::javaobject>("mBinding");
 
-  return getFieldValue(bindingField)->cthis();
+  return jni::static_ref_cast<Binding::javaobject>(getFieldValue(bindingField))->cthis();
```

Comment:
This is somewhat risky, if you do use a different Binding implementation, this cast will not be valid, but that's a pre-existing issue here.